src/components/ChatLog.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css'
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+    const chatLog = 'chat-log';
+    let showLike;
+    console.log(props.entries);
+    const chatComponents = props.entries.map((chat, index)=> {
+        const entriesData = [...props.entries];
+        const totalLike = [...entriesData].reduce((a,b) => a + (b.liked? 1: 0), 0)
```

Comment:
👍 Even though I'd recommend moving this logic, this is still a nice use of `reduce` to total up the number of liked messages in the chat.